### PR TITLE
Fix sidenav issue where is open when page loads. 

### PR DESCRIPTION
**Pull request by ERetana2**
Previous model had issue of sidenav being open when page loads. This PR addresses that issue

### CODE REVIEW

File: launchpod/src/main/webapp/angular-launchpod/src/app/navbar/navbar.component.html [hunk 1]
```diff
@@ -21,13 +21,14 @@ <h2>LaunchPod</h2>
...
-            <button type="button" aria-label="Toggle sidenav" mat-icon-button (click)="drawer.toggle()">
+            <button type="button" aria-label="Toggle sidenav" mat-icon-button (click)="drawer.open()">
                 <mat-icon aria-label="Side nav toggle icon">menu</mat-icon>
             </button>
             <img class="logo" src="assets/img/LaunchPodLogo.png" alt="icon">
             <span>LaunchPod</span>
         </mat-toolbar>
         <!-- Add Content Here -->
         <app-create-form class="center-page"></app-create-form>
+
```
Comment (bramanudom) [inline launchpod/src/main/webapp/angular-launchpod/src/app/navbar/navbar.component.html]: Nit: is this extra space?

File: launchpod/src/main/webapp/angular-launchpod/src/app/navbar/navbar.component.html [hunk 2]
```diff
@@ -21,13 +21,14 @@ <h2>LaunchPod</h2>
     </mat-sidenav>
     <mat-sidenav-content>
         <mat-toolbar color="white">
-            <button type="button" aria-label="Toggle sidenav" mat-icon-button (click)="drawer.toggle()">
+            <button type="button" aria-label="Toggle sidenav" mat-icon-button (click)="drawer.open()">
```
Comment (bramanudom) [inline launchpod/src/main/webapp/angular-launchpod/src/app/navbar/navbar.component.html:24]: Just for my understanding, why did you change this from toggle to open? Did you want the button to only ever open and not also close? (Sorry, if I'm misunderstanding)

Comment (ERetana2) [reply launchpod/src/main/webapp/angular-launchpod/src/app/navbar/navbar.component.html:24]: The toggle was for before when our content would be pushed to 'side,' but now with the 'over' property clicking outside of the sidenav closes it automatically so we only require of drawer.open property
